<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/image/images/backgrounds/MainScreen_Background_02.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushBtn_GType">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>134</width>
      <height>74</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>134</width>
      <height>74</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/images/Main Btns/Btns_Slices_09.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>290</y>
      <width>70</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>69</width>
      <height>69</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/images/Main Btns/Normal Btns/Btns_Slices_50.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>70</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>69</width>
      <height>69</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/images/Main Btns/Normal Btns/Btns_Slices_48.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>206</width>
      <height>260</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>206</width>
      <height>260</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/images/backgrounds/Background_Transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <action name="actionTest1">
   <property name="icon">
    <iconset>
     <selectedoff>:/image/Menus/1.jpg</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>test1</string>
   </property>
  </action>
  <action name="actionTest2">
   <property name="text">
    <string>test2</string>
   </property>
  </action>
  <action name="actionTest3">
   <property name="text">
    <string>test3</string>
   </property>
  </action>
  <action name="actionTest4">
   <property name="text">
    <string>test4</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
